assignments/objects.js

```diff
@@ -49,16 +95,42 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
```

Comment:
You definitely know what you're doing, the nesting of objects speak for itself and usage of "this" keyword.